docs/installation.md

```diff
@@ -1,8 +1,10 @@
 # Installation 
```

Comment:
Comment for line 34:

It says: "Include this code in your file". Which file? I think we must specify which file the user has to modify.

Reply:
Done!